src/test/java/LaunchingChrome_IT.java

```diff
@@ -0,0 +1,27 @@
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class LaunchingChrome_IT {
+    @Test
+    public void registration() {
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://www.saucedemo.com/");
+        WebElement elementFirstName = driver.findElement(By.id("user-name"));
```

Comment:
Pretty straight forward. No patters, no PageObjects.
Is that was expected?

Reply:
Actually 'Automation Best Practices & Patterns' - it's the next topic of course =D

- best practices of tests\page classes organization;
- types of automated tests;
- separation of concerns principle;
- why logs are important and how to implement logging.
- Page Object and Page Factory patterns;